### PR TITLE
editoast: update readme for onboarding

### PR DESCRIPTION
**Pull request by bougue-pe**
also follow-up on 'docker compose' propagation

### CODE REVIEW

File: editoast/README.md [hunk 1]
```diff
@@ -7,10 +7,13 @@ It will apply modification and update generated data such as object geometry.
 
 ## Requirements
 
+For both tests or run:
 - [rustup](https://rustup.rs/)
 - [libpq](https://www.postgresql.org/docs/current/libpq.html) (may be packaged as `libpq-dev`)
 - [openssl](https://www.openssl.org)
 - [libgeos](https://libgeos.org/usage/install/) (may be packaged as `libgeos-dev`)
+- A postgresql database (get inspiration from the [main docker-compose.yml](/docker-compose.yml), especially for database init)
```
Comment (shenriotpro) [inline editoast/README.md]: Why not give a `docker compose up ...` command, like in tests/?

**Review by shenriotpro — COMMENTED** [review]
Thanks

**Review by woshilapin — APPROVED** [review]

**Review by flomonster — APPROVED** [review]

File: editoast/README.md (hunk 1 above)
Comment (bougue-pe) [reply editoast/README.md]: Good idea indeed (didn't read that part), done in https://github.com/osrd-project/osrd/compare/c60304cbe08a710c0a7807c99a88a048dbfc64c0..a65d4f54681e6b31ecace2060cecc15612aaf132

File: editoast/README.md [hunk 2]
```diff
@@ -37,16 +40,17 @@ cargo test -- --test-threads=1
...
 
-Here a list of components to help you in your development:
+Here a list of components to help you in your development (see CI jobs if necessary):
 
- - [rustfmt](https://github.com/rust-lang/rustfmt): Format the whole code `cargo fmt`
- - [clippy](https://github.com/rust-lang/rust-clippy): Run a powerful linter `cargo clippy --all-features --all-targets -- -D warnings`
- - [tarpaulin](https://github.com/xd009642/tarpaulin): Check code coverage `cargo tarpaulin --skip-clean -o Lcov --output-dir target/tarpaulin/`
+- [rustfmt](https://github.com/rust-lang/rustfmt): Format the whole code `cargo fmt`
+- [clippy](https://github.com/rust-lang/rust-clippy): Run a powerful linter `cargo clippy --all-features --all-targets -- -D warnings`
+- [tarpaulin](https://github.com/xd009642/tarpaulin): Check code coverage `cargo tarpaulin --skip-clean -o Lcov --output-dir target/tarpaulin/`
```
Comment (woshilapin) [inline editoast/README.md]: I actually believe that `tarpaulin` has been removed from our stack of tools (in the [CI revamp](https://github.com/osrd-project/osrd/pull/5889)). Coverage results are now produced with `grcov`. See https://github.com/osrd-project/osrd/blob/5ac767c5ed078abc7a1c749219d5b3369e74e504/.github/workflows/build.yml#L354.

**Review by woshilapin — CHANGES_REQUESTED** [review]

File: editoast/README.md (hunk 2 above)
Comment (bougue-pe) [reply editoast/README.md]: Good catch, change done: https://github.com/osrd-project/osrd/compare/a65d4f54681e6b31ecace2060cecc15612aaf132..7fd21214b2d0fdd40196294cc35da7514ccc8b14

**Review by shenriotpro — COMMENTED** [review]
LGTM

**Review by woshilapin — APPROVED** [review]

**Review by shenriotpro — APPROVED** [review]

**Pull request merged**